### ISSUE TITLE
0.1.0-pre10, 0.1.0-rc.0, 0.1.0-rc.1, 0.1.0-rc.2 and 0.1.0-rc.3 have been released.

### ISSUE THREAD

**Issue by Happypig375** [verybadcat/CSharpMath#17]
It's time to have a new schedule. The old issue is getting hard to follow.
#### TextPainter
- [x] Accent commands
- [x] Update Typography
- [x] Correct non-brace argument parsing
- An implementation of special characters (if it is not done now, it will be a bad breaking change):
  - [x] #
  - [x] $
  - [x] %
  - [x] &
  - [x] \
  - [x] ^
  - [x] _
  - [x] {
  - [x] }
  - [x] ~
- [x] DrawOneLine

### To be in 0.1.0-rc.1
- [x] Fix unit tests of CSharpMath (core)
- [x] Fix \color with color name and 0x
- [x] Fix description box of Utils.NuGet

# The milestones below have been changed to after merger with wpf-math. Those shown are the original plan.

### To be in 0.1.0-rc.3
- [ ] Subpixel rendering of TextPainter (I am still unsure to whether include the whole PixelFarm just for this, gonna try though)

### Documentation before release
- [ ] Getting started with CSharpMath
- [ ] Changelog
- [ ] Documentation of MathPainter is unfinished on both NuGet and GitHub wiki
- [ ] [Extra docs on Measure](https://github.com/verybadcat/CSharpMath/issues/12#issuecomment-405013645)
- [ ] [2 new wiki pages](https://github.com/verybadcat/CSharpMath/issues/12#issuecomment-402373426)
- [ ] Release procedure is not documented

------------------

### Between 0.1.0 and 0.2.0
- Promotion of CSharpMath not yet done

------------------

### Will most probably be pushed to 0.2.0
- [ ] Implement \\middle
- `1 \\ { 2 \\ 3 }`'s spacing between 1 and 2 is weird
![image](https://user-images.githubusercontent.com/19922066/42049804-b52f8aba-7b39-11e8-8ae0-261fe49d7950.png)
- No way to limit scrolling to horizontal only or vertical only
- Loading times of CSharpMath.Forms.Android.Example is still very slow in Release mode
- `However, it is also not yet complete as in not being able to SeparateThisReallyLongWordWhichIsSoLongThatItSpansAcrossTheEntirePageAndWontStopEvenWhenPartOfItIsOutOfBounds.`
![image](https://user-images.githubusercontent.com/19922066/43326540-b3f37620-91eb-11e8-86e4-e40f3aef9d31.png)
### Will definitely be pushed to 0.2.0
- Refactor TextAtoms and MathAtoms to have common atoms
- Refactor MathAtoms to get rid of MathAtomType and AtomCloner
- Unit tests are missing in general for CSharpMath.Rendering
  (The unit tests will be dependent upon CSharpMath.SkiaSharp; and validated through picture comparisons)
- Tavis CI integration
- MathEditor port (#11)

Merged issue: #15

**Comment by charlesroddie** [comment]
NB nuget puts "0.1.0-pre10" between "0.1.0-pre1" and "0.1.0-pre2".
https://docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages
The next release/prerelease I would not call "0.1.0-pre11" therefore. Needs a increment to the verison number.

**Comment by Happypig375** [comment]
I re-read the SemVer homepage. I didn't know I had to put a dot between `pre` and `10` before, so I will do so next release. Thanks for pointing it out.

**Comment by Happypig375** [comment]
0.1.0-rc.1 has been released. Subpixel rendering looks to require extra work so it will get its own release candidate. Please report any issues.

**Comment by charlesroddie** [comment]
On rc.0 I am getting for TextPainter that @"\blue{a}" shows a in blue, but @"\blue{$a$}" has a in black.

**Comment by Happypig375** [comment]
Fixed on 0.1.0-rc.2. Subpixel will be in 0.1.0-rc.3. I don't want to move it to the backlog because it will greatly affect image-based unit tests later on.

**Comment by FoggyFinder** [comment]
@Happypig375 seems like there is a problem with colors. Text like this `blue{$x_{0}$}` doesn't render properly

**Comment by Happypig375** [comment]
0.1.0-rc.3 has been released. If there is no issues found within a few days, it will be the same as 0.1.0.

**Comment by Happypig375** [comment]
Superceded by #21.
